app/src/main/java/de/westnordost/streetcomplete/quests/place_name/AddPlaceName.kt

```diff
@@ -57,7 +57,10 @@ class AddPlaceName(
     override val commitMessage = "Determine place names"
     override val icon = R.drawable.ic_quest_label
 
-    override fun getTitle(tags: Map<String, String>) = R.string.quest_placeName_title_name
+    override fun getTitleBaseString(tags: Map<String, String>) = R.string.quest_placeName_title_name
+    override fun getTitleReplacements(tags: Map<String, String>, typeName: Lazy<String?>): Array<String?> {
+        return arrayOf(typeName.value)
+    }
```

Comment:
You can use the short-hand syntax here, so it is a one-liner.
```kotlin
override fun getTitleReplacements(tags: Map<String, String>, typeName: Lazy<String?>) = arrayOf(typeName.value)
```

Reply:
Is it OK to have long lines? In this case 130 characters.

Reply:
If it is still readable, it is okay.

Though, by the way, you could also leave away the `: Array<String?>`, because it is inferred